Remove ADV CPR

Tomcat-SG opened the pull request:
**When merged this pull request will:**
- Remove the dependency on Adv CPR and add settings to AED and AED-X
- Remove all known bugs related to kat

Tomcat-SG committed:
Remove ADV CPR

Tomcat-SG committed:
Fix Pulsoximeter Action and AEDX Condition

YetheSamartaka commented on addons/circulation/stringtable.xml:
```diff
@@ -312,6 +312,14 @@
+            <Polish>Użyj stacja AED</Polish>
+            <Spanish>Utilice el estación AED</Spanish>
+            <Chinese>使用AED除顫器</Chinese>
+			<Czech>Použít AED stanice</Czech>
```
this should be <Czech>Použít AED stanici</Czech>

Tomcat-SG replied:
Is in the new commit

Tomcat-SG committed:
Animation for patient and Bugfix

Tomcat-SG committed:
Make the AED as placed Item usable for all in the near

Tomcat-SG committed:
fix setDead call

Tomcat-SG committed:
remove nonsense code

Tomcat-SG committed:
remove the setUnconscious for low spo2

gets removed to get space for fresh new code which is using the state machine to make an transition

Tomcat-SG committed:
Update fnc_adjustSpo2.sqf

fix getting unconscious from spo2 value

Tomcat-SG commented:
All Animations need to be tested

Tomcat-SG committed:
fix an comparison error

YetheSamartaka committed:
stringtable update

stringtable update for:
misc
circulation
breathing
airway

Tomcat-SG committed:
Merge pull request #21 from YetheSamartaka/removeAdvCPR

stringtable update

Tomcat-SG committed:
Update fnc_treatmentAdvanced_airwayLocal.sqf

triggered me xD

Tomcat-SG committed:
Update fnc_adjustSpo2.sqf

fix too much spo2

Tomcat-SG committed:
Update fnc_init.sqf

only for a little time as an workaround till the rewirte of adjustment of spo2

YetheSamartaka committed:
Temporary AED-X monitor workaround

This will add option to deactivate Monitor while using AED-X.

Tomcat-SG committed:
Update ACE_Medical_Treatment_Actions.hpp

Tomcat-SG committed:
Merge pull request #22 from YetheSamartaka/removeAdvCPR

Temporary AED-X monitor workaround

Tomcat-SG committed:
remove hr while aed (if setting true)

Tomcat-SG committed:
Update fnc_treatment.sqf

fix patient animation

Tomcat-SG committed:
remove debug and fix animation

Tomcat-SG committed:
Update fnc_treatment.sqf

Tomcat-SG committed:
add heal of airway damage

Tomcat-SG committed:
add overtretch treatmentTime setting

Tomcat-SG commented:
We need only a solution for the animations and after that, this update is done

Tomcat-SG committed:
add setting

forgot to push the preInit change

Tomcat-SG committed:
remove duplicated aedx behaviour

Tomcat-SG committed:
change aed position
